install/tools/certs/Makefile

```diff
@@ -12,22 +14,22 @@ ROOTCA_DAYS ?= 3650
 ROOTCA_KEYSZ ?= 4096
 ROOTCA_ORG ?= Istio
 ROOTCA_CN ?= Root CA
+KUBECONFIG ?= $(HOME)/.kube/config
 # Additional variables are defined in root-ca.conf target below.
 
 #------------------------------------------------------------------------
-# variables: intermediate CA (Citadel)
-CITADEL_SERIAL ?= $(shell echo $$PPID) 	# certificate serial number (uses current PID)
-CITADEL_DAYS ?= 730
-CITADEL_KEYSZ ?= 4096
-CITADEL_ORG ?= Istio
-CITADEL_CN ?= Intermediate CA
-CITADEL_SAN_DNS ?= localhost
+# variables: intermediate CA
+INTERMEDIATE_SERIAL ?= $(shell echo $$PPID) 	# certificate serial number (uses current PID)
```

Comment:
You may want to read from random. 
If I remember correctly some openssl-based-CA had some way to save the serial in a file, and update it on each new generated cert. 

Not a big deal for testing/demo - but this makefile is looking closer to a prod-usable automation tool.

Reply:
ya, the serial numbers will auto-increment when openssl does anything with them. No real need to store this file. Using the PPID is not ideal, as it may be used twice for different certs.